Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestDetailsValidationService.cs

```diff
@@ -6,16 +6,15 @@
 
 public class ContestDetailsValidationService : IContestDetailsValidationService
 {
-    public ValidationResult GetValidationResult((Contest, int?, UserInfoModel?) item)
+    public ValidationResult GetValidationResult((Contest, int?, bool) item)
     {
-        var (contest, contestId, user) = item;
+        var (contest, contestId, isUserAdminOrLecturerInContest) = item;
```

Comment:
ContestId is passed since we need it for displaying message if GetValidationResult returns Invalid. When passing the contest to GetValidationResult if contest is null then contestId will not be accessible for the NotFound message